privacy_partner_report/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+======================
+Privacy Partner Report
+======================
+
+This module helps a company to identify all the transactions that a specific
+partner is involved in, with the possibility to export the associated data.
+
+
+Usage
+=====
+
```

Comment:
Could you please describe usage?

I have gone to a Partner and I don't see any option to get any report. Is it possible to add from res.partner form?

Thanks! :+1:

Reply:
Done :+1: